test/powershell/engine/Module/TestModuleManifest.Tests.ps1

```diff
@@ -274,3 +274,24 @@ Describe "Test-ModuleManifest Performance bug followup" -tags "CI" {
         }
     }
 }
+
+Describe 'Testing module with wildcard export' -Tags 'CI' {
+    BeforeAll {
+        $modulePath = 'testdrive:/wildcardexport'
+        New-Item -ItemType Directory -Path $modulePath > $null
+    }
+
+    AfterAll {
+        Remove-Item -Recurse -Force -ErrorAction SilentlyContinue testdrive:/module
+    }
+
+    It 'PSModuleInfo does not accumulate tags' {
+        # https://github.com/PowerShell/PowerShell/issues/17694
+        New-Item -ItemType File -Path "$modulePath\tags.psm1" > $null
+        New-ModuleManifest -Path "$modulePath\tags.psd1" -RootModule 'tags.psm1' -FunctionsToExport '*' -Tags 'one'
```

Comment:
should add a test case for case-sensitivity.  I would think `-Tags 'one','ONE'` would only result in 1 tag.